test/adapter/json/polymorphic_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class Json
+        class PolymorphicTest < ActiveSupport::TestCase
+          def setup
```

Comment:
Now I'm with you. Thanks for the hand-holding (and learning me something new)! Updated.